library/common-library/DEVELOPERS.md

```diff
@@ -99,6 +99,37 @@ spec:
 
 
 
+## _annotations.tpl
+### `newrelic.common.annotations.deployment` and `newrelic.common.annotations.pod`
+These are functions that are used to annotate objects. They are configured by this `values.yaml`
+```yaml
+global:
+  deploymentAnnotations: {}  # included in all the deployments and daemonsets of all the charts
```

Comment:
having deployment annotations added for daemonsets is counterintuitive. Could we name it in a different way?

Reply:
We can change it to `deployerAnnotations` as I suggested above.

Reply:
Let's focus on one thread, please because the issue is the same.

Let's do the follow-up in [this thread](https://github.com/newrelic/helm-charts/pull/802#discussion_r865917563).